src/StackInterface.php

```diff
@@ -43,5 +17,5 @@ public function withoutMiddleware(MiddlewareInterface $middleware);
      *
      * @return ResponseInterface
      */
-    public function process(RequestInterface $request);
+    public function dispatch(RequestInterface $request);
```

Comment:
@mnapoli yes I did go through the groups discussion and issue. Probably I didn't quite understand whether change in method name have any benefit. I will go though it once again to understand the reasoning.